packages/core/src/exports.ts

```diff
@@ -208,6 +209,15 @@ export function withScope<T>(
   return getCurrentHub().withScope(rest[0]);
 }
 
+/**
+ * TODO
```

Comment:
I think there's still something missing 😅

Reply:
Thank you!